Makefile

```
@@ -1,8 +1,13 @@
 FEATURES ?= vtpm
+ifneq ($(FEATURES),)
 SVSM_ARGS += --features ${FEATURES}
+endif
```

Since we are touching it, I was thinking on something similar, but supporting also `--no-default-feature`. I mean when FEATURES is empty, we can build without defaults.

I mean something like we have for tests:
```suggestion
FEATURES ?= default,vtpm
SVSM_ARGS += --no-default-features
ifneq ($(FEATURES),)
SVSM_ARGS += --features ${FEATURES}
endif
```

WDYT?

---

I think as it is the user can already do this by setting `SVSM_ARGS=--no-default-features` from the command line, which is more intuitive, as default features should be opt-out.

---

yep, right, let's not overcomplicate it.